Doc/whatsnew/3.10.rst

```diff
@@ -1774,6 +1774,16 @@ Build Changes
 C API Changes
 =============
 
+PEP 652: Maintaining the Stable ABI
+-----------------------------------
+
+The Stable ABI (Application Binary Interface) for extension modules or
```

Comment:
Hm. The Stable ABI and C API in here?

Reply:
Well, the limited C API is also defined. I went with the PEP name to make things shorter.